operator/webhooks/pod_mutator/pod_mutating_webhook.go

```diff
@@ -441,7 +450,7 @@ func (a *PodMutatingWebhook) getAppName(pod *corev1.Pod) string {
 	return strings.ToLower(applicationName)
 }
 
-func (a *PodMutatingWebhook) getOwnerReference(resource *metav1.ObjectMeta) metav1.OwnerReference {
+func (a *PodMutatingWebhook) getOwnerReference(resource metav1.ObjectMeta) metav1.OwnerReference {
 	reference := metav1.OwnerReference{}
 	if len(resource.OwnerReferences) != 0 {
 		for _, owner := range resource.OwnerReferences {
```

Comment:
Maybe to keep things more easily aligned, we could create a common function that returns if a Kind is supported. This function then can be used here and in the operator logic.